core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -580,3 +580,89 @@
     </li>
   </ul>
 </div>
+<style>
+  .dropdn {
+    position: absolute;
+    z-index: 500;
+  }
+  @media only screen and (max-width: 335px) {
+  :lang(ar) .sinin {
+font-size: 0.5rem !important;
+padding: 0;
+position: relative;
+top: 2rem;
+  }
+  }
+
+    @media only screen and (max-width: 355px) {
+    .langbtn {
+    margin-left: 0;
+    margin-right: 0;
+    margin-top: 4px;
+    padding: 0;
+  }
+  .langi {
+    margin-left: 0;
+    margin-right: 0;
+    margin-top: 4px;
+    padding-left: 2px;
+  }
+   .langcd {
+    margin-left: 0;
+    margin-right: 0;
+    margin-top: 4px;
+    padding: 0;
+  }
+    .langbtnContainer {
+    margin-left: 0;
+    margin-right: 0;
+    margin-top: 4px;
+  }
+   .sinin {
+     margin-left: 0;
+     margin-right: 0;
+     margin-top: 3px;
+     padding: 3px;
+  }
+  :lang(ar) .langbtn {
+    margin-top: 6px !important;
```

Comment:
Hey @Ykumar1415 , I am not sure using the !important flag is really recommended. And the same applies for the usage of the flag below as well. Could you find an alternate solution? Thanks!

Reply:
> @SanjaySajuJacob i have removed the !important flag on this place without it also working fine. The other thing is that the button in arabic language is specifically can only be accessed with :lang() because i not only solving this issue also there was many extra problems with this page in arabic language i also targeting then it only be possible for me with the help of specific lang selector.I have also search on google but there is not problem i found in using this.

I will keep this remember. Thanks.